cypress/integration/sample_spec.js

```diff
@@ -59,7 +59,7 @@ describe('Login / Register', () => {
             cy.get('#psw').type('secretPassword123')
             cy.get('#login-button').click()
             
-            cy.contains('#loing-error').should('not.exist')
+            cy.contains('#loging-error').should('not.exist')
```

Comment:
#login-error not #loging-error

Reply:
This should be commented in PR #44 since the change was done there (fixed now). It might be a good idea to delay the review until PR #44 is added because the diffs are not accurate